### PR TITLE
Document SR-IOV configuration

### PR DESCRIPTION
**Pull request by booxter**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: playbooks/README.md [hunk 1]
```diff
@@ -169,6 +169,52 @@ $ ansible-playbook -i inventory -e "selinux=permissive" playbooks/selinux.yml
...
+
+SR-IOV device plugin uses ```/etc/pcidp/config.json``` file to determine which
+PFs should be used to allocate VFs from. By default, ```kubevirt-ansible```
+doesn't configure any PFs, meaning that the plugin has zero devices to manage.
+
+There are two ways to configure allocatable PFs:
+
+a) use ```sriov_pci_ids``` playbook variable to specify PFs, for example:
+
+$ ansible-playbook -i inventory -e "sriov_pci_ids=\"0000:05:00.0\",\"0000:05:00.1\"" playbooks/kubevirt.yaml
```
Comment (SchSeba) [inline playbooks/README.md]: please add -e@vars/all.yml

**Comment by booxter** [comment]
@SchSeba I've pushed an update addressing your comment.

**Comment by myakove** [comment]
ci test please

**Comment by myakove** [comment]
LGTM

**Comment by booxter** [comment]
@rthallisey could you please review this documentation change for SR-IOV?

File: playbooks/README.md [hunk 2]
```diff
@@ -169,6 +169,52 @@ $ ansible-playbook -i inventory -e "selinux=permissive" playbooks/selinux.yml
 
 Be sure to update the [inventory file](../inventory) according to your OpenShift cluster configuration or use the file you used to deploy the cluster.
 
+### SR-IOV
```
Comment (rthallisey) [inline playbooks/README.md]: ### Networking
Kubevirt-ansible enables sriov by default, using the kubernetes CNI plugin multus.
```yaml
# vars/all.yml
...
deploy_sriov_plugin: true
```

For sriov to work properly, configure your hosts to enable IOMMU, SR-IOV in the BIOS and kernel and vfio-pci in the kernel.   Also, enable the expected number of VFs before starting.

File: playbooks/README.md [hunk 3]
```diff
@@ -169,6 +169,52 @@ $ ansible-playbook -i inventory -e "selinux=permissive" playbooks/selinux.yml
...
+
+There are two ways to configure allocatable PFs:
+
+a) use ```sriov_pci_ids``` playbook variable to specify PFs, for example:
+
+$ ansible-playbook -i inventory -e@vars/all.yml -e "sriov_pci_ids=\"0000:05:00.0\",\"0000:05:00.1\"" playbooks/kubevirt.yaml
+
+Note: this won't work if you have different PCI IDs on different nodes.
+
+b) deploy Kubevirt as usual, then manually edit ```/etc/pcidp/config.json``` on
```
Comment (phoracek) [inline playbooks/README.md]: with the configmap approach, changes to this file will be overwritten during sriov dp restart with configmap content. we can either change the initContainer to keep existing config.json or edit this documentation

Comment (booxter) [reply playbooks/README.md]: Indeed I should now document ConfigMap.

File: playbooks/README.md (hunk 2 above)
Comment (booxter) [reply playbooks/README.md]: @rthallisey please help me here, I am trying to understand if what you posted is a suggestion to replace some of the text below verbatim? Or is it something else?

**Comment by booxter** [comment]
@rthallisey Could you please check the latest version and clarify your comment? Thanks.

File: playbooks/README.md (hunk 2 above)
Comment (rthallisey) [reply playbooks/README.md]: It was a suggestion to replace some text here in the in the README.

**Comment by rthallisey** [comment]
@booxter sorry I didn't get back to you sooner.

**Comment by booxter** [comment]
@rthallisey please have a look, I've adopted your wording suggestion (with minor modifications).

**Comment by rthallisey** [comment]
@booxter just needs a rebase.  LG2M!

**Comment by booxter** [comment]
Rebased. (This is weird that I should apparently rebase before each merge. I thought git has branches and merges.)

**Comment by rthallisey** [comment]
It's only suppose to block on conflicts; fixed the setting.

**Pull request merged**